### PR TITLE
Minor edits for readability improvements - Environment variables

### PR DESCRIPTION
**Pull request by sarah11918**
Some edits in text, some for cosmetic purposes because there's a lot going on with all the bits of code close together and all the `import[dot]meta[dot]env`.

### CODE REVIEW

**Comment by Yan-Thomas** [comment]
Looks good to me, great work!

File: src/pages/en/guides/environment-variables.md [hunk 1]
```diff
@@ -6,12 +6,10 @@ description: Learn how to use environment variables in an Astro project.
 
 Astro uses Vite for environment variables, and allows you to use any of its methods to get and set environment variables. 
 
-Note that while _all_ environment variables are available in server-side code, only environment variables prefixed with `PUBLIC_` are avilable in client-side code.
+Note that while _all_ environment variables are available in server-side code, only environment variables prefixed with `PUBLIC_` are avilable in client-side code for security purposes.
```
Comment (jcha0713) [inline src/pages/en/guides/environment-variables.md:9]: Found a typo: avilable -> available

Comment (sarah11918) [inline src/pages/en/guides/environment-variables.md:9]:
```suggestion
Note that while _all_ environment variables are available in server-side code, only environment variables prefixed with `PUBLIC_` are available in client-side code for security purposes.
```

Comment (sarah11918) [reply src/pages/en/guides/environment-variables.md]: Fixed! Thanks @jcha0713 ! 🥳

**Comment by sarah11918** [comment]
Thanks @jcha0713 and @Yan-Thomas !

**Pull request merged**